code/game/objects/items/tools/wirecutters.dm

```diff
@@ -20,14 +21,34 @@
 	toolspeed = 1
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 30)
 	var/random_color = TRUE
+	var/static/list/wirecutter_colors = list(
+		"blue" = rgb(24, 97, 213),
+		"red" = rgb(149, 23, 16),
+		"pink" = rgb(213, 24, 141),
+		"brown" = rgb(160, 82, 18),
+		"green" = rgb(14, 127, 27),
+		"cyan" = rgb(24, 162, 213),
+		"yellow" = rgb(213, 140, 24)
+	)
+
 
+/obj/item/wirecutters/Initialize()
+	. = ..()
+	if(random_color) //random colors!
+		icon_state = "cutters"
+		var/our_color = pick(wirecutter_colors)
+		add_atom_colour(wirecutter_colors[our_color], FIXED_COLOUR_PRIORITY)
+		update_icon()
+	if(prob(75))

```

Comment:
what's this?

Reply:
iirc this was just to spread them out if they're on top of each other

Reply:
yes, but why are you adding it?

Reply:
dunno, this is 5 month old code